index.html

```diff
@@ -36,118 +36,116 @@ <h2>Stages</h2>
       <ul>
-        <li>Make the case for the addition
-        <li>Describe the shape of a solution
+        <li>Make the case for an improvement
+        <li>Describe the shape of some possible solutions
         <li>Identify potential challenges
+        <li>Research how the problem is dealt with using available facilities today
+        <li>Research how the problem has been solved by other languages or in the library ecosystem
       </ul>
-    </td>
+  </tr>
+
+  <tr>
+    <td>1
+    <td>This proposal is under consideration. The committee expects to devote time to examining the identified problem space, the full breadth of possible solutions, and cross-cutting concerns. 
     <td>
       <ul>
-        <li>Identified “champion” who will advance the addition
+        <li>Identified a champion or champion group who will advance the addition
         <li>Prose outlining the problem or need and the general shape of a solution
-        <li>Illustrative examples of usage
-        <li>High-level API
         <li>Discussion of key algorithms, abstractions and semantics
-        <li>Identification of potential “cross-cutting” concerns and implementation challenges/complexity
+        <li>Identification of potential cross-cutting concerns and implementation challenges/complexity
         <li>A publicly available repository for the proposal that captures the above requirements
       </ul>
     </td>
-    <td>The committee expects to devote time to examining the problem space, solutions and cross-cutting concerns
+    <td>
+      Designing a solution.
+      <ul>
+        <li>Make the case for a particular solution or solution space
+        <li>Resolve any cross-cutting concerns
+      </ul>
     </td>
-    <td>None
-    <td>Major
-    <td>Polyfills / demos
   </tr>
+
   <tr>
     <td>2
-    <td>Precisely describe the syntax and semantics using formal spec language
+    <td>The committee has chosen a preferred solution or solution space, but the design is a draft and may still change significantly. The committee expects the feature to be developed and eventually included in the standard, but due to reasons that may not be apparent at this stage, the feature may never advance.
     <td>
       <ul>
-        <li>Above
-        <li>Initial spec text
+        <li>Proposal document describes all high-level APIs and syntax
+        <li>Illustrative examples of usage
+        <li>Initial spec text including all major semantics, syntax, and APIs. Placeholders, TODOs, and editorial issues are acceptable
+      </ul>
+    </td>
+    <td>
+      Refining the solution.
+      <ul>
+        <li>Work out minor details such as ordering of observable effects, handling of invalid inputs, API names, etc.
+        <li>Receive and address spec text reviews from the assigned reviewers and the appropriate editor group
+        <li>Produce experimental implementations such as loosely-correct polyfills to aid in validating the design and exploring the details
```

Comment:
```suggestion
        <li>Produce experimental implementations such as loosely-correct non-production-usage polyfills to aid in validating the design and exploring the details
```

Reply:
Accepted as "loosely-correct (not for production use) polyfills".

Reply:
That captures what I wanted, thanks :-)